Seohee/Lv2/다리를 지나는 트럭.js

```diff
@@ -0,0 +1,33 @@
+function solution(bridge_length, weight, truck_weights) {
+    // 다리위의 트럭
+    let onBridge = [];
+    let onBridgeTime = [];
+    //총시간
+    let time = 1;
+    // 무게 합산
+    let sum = truck_weights[0];
+    //맨처음 트럭 가져오고 시간 초기화
+    onBridge.push(truck_weights.shift())
+    onBridgeTime.push(1);
+    // 다리 위가 빌때까지 실행
+    while (onBridge.length != 0) {
+        //맨앞에 있는 트럭이 다리를 지나면 무게를 빼주고, 다리위에서 트럭을 제거
+        if (onBridgeTime[0] === bridge_length) {
+            sum -= onBridge[0];
+            onBridge.shift();
+            onBridgeTime.shift();
+        }
+        // 다리 위에 있는 각각의 트럭에 대해 시간 1씩 증가
+        onBridgeTime = onBridgeTime.map((data) => data + 1);
```

Comment:
```suggestion
        onBridgeTime.forEach((_, idx) => onBridgeTime[idx] += 1);
```
새로운 배열이 필요한게 아니다보니까 map보다는 forEach를 사용하는게 적절해보이고 이러면 onBridgeTime을 const로 선언할 수 있습니다.